src/api/flaskr/api/tts/baidu_provider.py

```
@@ -0,0 +1,930 @@
+        "client_id": api_key,
+        "client_secret": secret_key,
+    }
+
+    try:
+        response = requests.post(BAIDU_TOKEN_URL, params=params, timeout=30)
+        response.raise_for_status()
+        result = response.json()
+
+        if "access_token" not in result:
+            error_msg = result.get("error_description", "Unknown error")
+            raise ValueError(f"Failed to get Baidu access token: {error_msg}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)